### ISSUE TITLE
Stepping into code for Grpc.* packages when debugging doesn't work

### ISSUE THREAD

**Issue by JamesNK** [grpc/grpc#18287]
### What version of gRPC and what language are you using?
C#
 
Grpc.* is using `SourceLink.Create.CommandLine` for Source Link. That is an old package that has been replaced with `Microsoft.SourceLink.*` packages.

More info: https://github.com/ctaggart/SourceLink

This is the modern package: https://www.nuget.org/packages/Microsoft.SourceLink.GitHub

**Comment by jtattermusch** [comment]
We're using https://github.com/grpc/grpc/blob/master/src/csharp/Grpc.Core/SourceLink.csproj.include which looks like stepping into the code when debugging should work. Is that not the case for you?
If stepping into the code is broken, we should fix asap. If it works, this item is not a high priority.

**Comment by JamesNK** [comment]
I tested and Source Link does work with the Grpc.* packages.

I tested Grpc.Core.Api and Newtonsoft.Json in the same application. They both have PDBs in the primary NuGet package.

Stepping into Grpc.Core.Api results in this:

![image](https://user-images.githubusercontent.com/303201/53939204-45a4eb00-4117-11e9-91e7-9531fe5235c3.png)

Stepping into Newtonsoft.Json fetches the file and loads it from the SourceServer cache on disk:

![image](https://user-images.githubusercontent.com/303201/53939221-4fc6e980-4117-11e9-86db-fd29a5559924.png)
